measurements-to-gelf.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+
+#
+# Prototype of a RIPE Atlas to GELF parser by awlx 
+#
+# Usage: python measurements-to-gelf.py <id> <min>
+#
+
+# Imports
+from gelfclient import UdpClient
+from urllib2 import urlopen
+import syslog
+import json
+import time
+import sys
+import requests
+
+# Configuration
+gelf_server = 'localhost'
+gelf_port = 5555
+api_key = '<api_key>'
+
+# Parse Args
+if len(sys.argv) == 1:
+    print "# Prototype of a RIPE Atlas to GELF parser"
+    print "#"
+    print "# Usage: python measurements-to-gelf.py <measurement id> <timeframe in min>"
+    print "#"
+    print "# Example: python measurements-to-gelf.py 12323 5"
+    print "#"
+    print "# Please define Server and Port inside the script and you need an API Key for Geolocation from https://geocoder.opencagedata.com/pricing"
+    exit(1) 
+
+measurement_id = str(sys.argv[1])
+timeframe_secs = int(sys.argv[2]) * 60
+
+
+
+def getplace(lat, lon):
+    url = "http://api.opencagedata.com/geocode/v1/json?q="
+    url += "%s+%s&key=%s" % (lat, lon, api_key)
+    v = urlopen(url).read()
```

Comment:
one-letter variables like `v` and `j` are responsible for the deaths of millions of kittens every year.  Please, think about the kittens and use whole words when writing code.  Future developers will thank you for being explicit ;-)